extensions/data-transfer/portability-data-transfer-backblaze/src/test/java/org/datatransferproject/datatransfer/backblaze/photos/BackblazePhotosImporterTest.java

```diff
@@ -42,119 +43,120 @@
 import org.datatransferproject.types.common.models.photos.PhotoModel;
 import org.datatransferproject.types.common.models.photos.PhotosContainerResource;
 import org.datatransferproject.types.transfer.auth.TokenSecretAuthData;
-import org.junit.Before;
```

Comment:
would you mind fixing the whitespace so it's not all reformatted? (or if we want to do that, then do it in a separate PR)

That is: I see it switches in this file from 2 char indentation to 4 char, but this makes your PR's real changes (junit migration) more difficult to follow

Reply:
> (or if we want to do that, then do it in a separate PR)

also on this^ front: Will and I have discussed automating style choices for the codebase, so it's probably not worth bothering here as it's on our future-hackathon-TODO list to tackle more holistically

Reply:
> would you mind fixing the whitespace so it's not all reformatted? (or if we want to do that, then do it in a separate PR)
> 
> That is: I see it switches in this file from 2 char indentation to 4 char, but this makes your PR's real changes (junit migration) more difficult to follow

Sure will keep that in mind, Please let me know if we still find it difficult to follow after toggling this "Hide whitespace" checkbox on diff view from the settings icon, In that case i'll fix it to the previous formatting and reapply all the changes.

<img width="306" alt="Screenshot 2022-08-31 at 4 12 04 pm" src="https://user-images.githubusercontent.com/13625776/187716063-0218a97b-5355-441c-9535-8d8d4e2c5695.png">